editing_and_imputation/editing/totals_components/technical_specification.md

```diff
@@ -0,0 +1,291 @@
+
+Else we move onto the next stage which is where we determine an error detection method.
+
+### 5.4 Determine Error Detection Method (Stage 5)
+
+If 
+
+```
+    x_{absolute} ≠ None
+```
+
+ we go to stage 5a found below. Else we go to section 5.4.3.
+
+### 5.4.1 Check Absolute Difference Threshold (Stage 5a)
+
+The absolute difference between the target period total and the components less than or equal to the absolute difference threshold can be visualised using the following formula. 
+
+```
+    x_{absolute} <= |y_{total, t} - y_{total, predictive}|
+```
+
+If this is true we mark 
+
+```
+    satisfied = TRUE
+``` 
+
+else we mark 
+
+```
+    satisfied = FALSE
+```
+
+we return the satisfied boolean and go to stage 6 in section 5.5 (if boolean is ```TRUE```), if boolean is ```FALSE``` we move to section 5.4.3.
+
+Note: Before we leave this stage this point we need to check that zero error condition 3 is satisfied
+
+```
+    y_{total, t} = 0
+```
+
+,
+
+```
+y_{derived, t} > 0
+```
+
+and amend total = ```True```.
+
+If so apply the correction to override the components with zeros if the difference observed is within the tolerances determined by the detection method.
+
+We then set TCC = M for manual checking.
+### 5.4.2 Check Percentage Difference Threshold (Stage 5b)
+
+If the total for the predictive period is within the low and high percentage as shown by the formula below 
+
+```
+    y_{derived} - (y_{derived} * x_{percent}) =< y_{total, predictive} =< y_{derived} + (y_{derived} * x_{percent})
+```
+
+If this is true we mark 
+
+```
+    satisfied = TRUE
+``` 
+
+else we mark 
+
+```
+    satisfied = FALSE
+```
+
+we return the satisfied boolean and go to stage 6 in section 5.5 (if boolean is ```TRUE```), if boolean is ```FALSE``` we return a TCC = M marker meaning manual editing is required and write an output.
+
+### 5.4.3 Percentage Difference Threshold
+
+This stage is to check if 
+
+```
+    x_{percent} ≠ None
+```
+
+If this is true we run stage 5b (see section 5.4.2 above), else we check that
+
+```
+    x_{absolute} ≠ None
+```
+or
+
+```
+    x_{percent} ≠ None
+```
+
+If it is true we return a TCC = M marker meaning manual editing is required and we write an output.
+
+### 5.5 Error Correction (Stage 6)
+
+This section covers the error correction aspect of the process.
+
+At this point we rely on the amend value `do_amend_total`. This is where `y_derived` or `y_total` are > 0.
+
+If 
+
+```
+    do_amend_total = TRUE
+```
+
+Then the total is automatically corrected for the target period to equal the sum of the components in the target period.
+
+Hence we set the following
+
+```
+    y_{derived} = y_{total, t}
+```
+
+and the final values for all the components match their originals. 
+
+We would then have a TCC = T marker and write the output.
+
+However, if
+
+```
+    do_amend_total = FALSE
+```
+
+Then the components are automatically corrected for the target period to equal the total in the target period.
```

Comment:
Can you say that the components are corrected to match the received predicted total based on the weighting of the original input component values.